apps/client/src/components/settingsMenu/settingsMenu.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useState } from 'react';
+import styles from './settingsMenu.module.css';
+import { Dialog, Switch, Tab } from '@mui/material';
+import { TabPanel, TabContext, TabList } from '@mui/lab';
+import {
+  useEditMutation,
+  useGetUsersQuery,
+  useLoginMutation,
+} from '../../store';
+import { User } from '../../types/user.types';
+
+interface SettingsMenuProps {
+  setIsOpenSettingsModal: React.Dispatch<React.SetStateAction<boolean>>;
+  isOpenSettingsModal: boolean;
+}
+
+export const SettingsMenu: React.FC<SettingsMenuProps> = ({
+  isOpenSettingsModal,
+  setIsOpenSettingsModal,
+}) => {
+  const [username, setUsername] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [password, setPassword] = useState('');

```

Comment:
I think `useState('')` shouldn't contain an empty string as its falsish. I believe it should be `useState()`.